linode_api4/linode_client.py

```diff
@@ -1433,6 +1433,513 @@ def cancel(self):
         self.client.post("/object-storage/cancel", data={})
         return True
 
+    def transfer(self):
+        """
+        The amount of outbound data transfer used by your account’s Object Storage buckets,
+        in bytes, for the current month’s billing cycle. Object Storage adds 1 terabyte
+        of outbound data transfer to your data transfer pool.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-transfer-view
+
+        :returns: The amount of outbound data transfer used by your account’s Object
+                  Storage buckets, in bytes, for the current month’s billing cycle.
+        :rtype: integer
```

Comment:
It actually returns a MappedObject instead of an integer. You can say something like this:

```suggestion
        :rtype: MappedObject contains an integer named `used`
```

Reply:
It's really not returning an `int` here...

Reply:
fixed.